qa/L0_perf_nomodel/test.sh

```diff
@@ -139,7 +139,7 @@ TEST_CONCURRENCY+=(
     1
     16
     16)
-TEST_BACKENDS="plan custom graphdef savedmodel onnx libtorch python"
+TEST_BACKENDS="custom"
```

Comment:
Don't you want to run nomodel test on all backends?

Reply:
Fixing, missed this in the cleanup